### PR TITLE
Change the template to use human readable task_instance description

### PR DESCRIPTION
**Pull request by potiuk**
The task_instance_key_str description is a better template
example as a human-readable description of the task.

---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

File: airflow/example_dags/example_bash_operator.py [hunk 1]
```diff
@@ -58,7 +58,7 @@
     # [START howto_operator_bash_template]
     also_run_this = BashOperator(
         task_id='also_run_this',
-        bash_command='echo "run_id={{ run_id }} | dag_run={{ dag_run }}"',
+        bash_command='echo "run_id={{ task_instance_key_str }}"',
```
Comment (jedcunningham) [inline airflow/example_dags/example_bash_operator.py]:
```suggestion
        bash_command='echo "ti_key={{ task_instance_key_str }}"',
```

Maybe?

Comment (potiuk) [reply airflow/example_dags/example_bash_operator.py]: Obviously :D

**Comment by potiuk** [comment]
@jedcunningham  ?

**Review by jedcunningham — APPROVED** [review]

**Comment by jedcunningham** [comment]
(Oops, sorry, missed this)

**Pull request merged**